modules/question-answering/data.tf

```diff
@@ -146,7 +151,7 @@ data "aws_iam_policy_document" "question_answering" {
     content {
       sid = "OpenSearch"
 
-      actions = statement.value.actions
+      actions = [statement.value.actions]
```

Comment:
why did you make this change? I'm getting an error here, but it doesn't flag when it's `actions = statement.value.actions`

Reply:
Sorry about that.